internal/gitutil/git.go

```diff
@@ -72,10 +72,11 @@ func EnsureUpdated(uri, destinationPath string) error {
 
 // GetRemoteURL returns the url of the remote origin
 func GetRemoteURL(dir string) (string, error) {
-	return exec(dir, "config", "--get", "remote.origin.url")
+	remote, err := Exec(dir, "config", "--get", "remote.origin.url")
+	return strings.TrimRight(remote, "\n\r"), err
```

Comment:
You can use strings.TrimFunc(unicode.IsSpace)

Furthermore, maybe just move this to Exec, as other cmds are likely going to need it.

Reply:
Ended up using `strings.TrimSpace`, the linter complained about `strings.TrimFunc('...', unicode.IsSpace)`